Source/WebKit/UIProcess/API/C/playstation/WKView.cpp

```diff
@@ -56,7 +56,20 @@ WKCursorType toWKCursorType(const WebCore::Cursor& cursor)
 
 WKViewRef WKViewCreate(WKPageConfigurationRef configuration)
 {
+#if USE(WPE_BACKEND_PLAYSTATION)
+    RELEASE_ASSERT_WITH_MESSAGE(false, "API unavailable with WPE Backend PlayStation");
+#else
     return WebKit::toAPI(WebKit::PlayStationWebView::create(*WebKit::toImpl(configuration)).leakRef());
```

Comment:
@fujii the `leakRef` is coming from here.

Reply:
WK API convention is that not leaking here.

Reply:
`WebKit::PlayStationWebView::create` is returning a `RefPtr` so it appears this is the way.